docs/EthStats/Lite-Network-Health-Monitor.md

```diff
@@ -0,0 +1,49 @@
+description: Lite Network Health Monitor
+<!--- END of page meta data -->
+
+# Lite Network Health Monitor
+
+This is a tool that helps monitor the health of a private network by displaying real time and historical statistics 
+from the perspective of its nodes.
+
+The lite version will persist either in memory or Redis only a fixed number of blocks, by default the last 3000. The full version requires PostgreSQL or Cassandra as persistence storage and has a more complex configuration to run its services in cluster mode.
+
+The tool can be seen in action for the Ethereum mainnet network at [net.ethstats.io](https://net.ethstats.io)
+
+## Statistics Overview
+* List of nodes in the network with specific metrics regarding its last received block like number, hash, transaction and uncle count, block time and propagation time and other metrics like connected peers, if it's mining or not, hash rate, latency, uptime.
+* Charts for Block Time, Block Difficulty, Block Gas Limit, Block Uncles, Block Transactions, Block Gas Used, Block Propagation Histogram and Top miners
+* IP based geolocation overview
+* Node logs (displays the data sent by a node)
+* Block history (ability to go back in time and playback the block propagation throughout the nodes)
+
+## Github repositories
+* [ethstats-cli](https://github.com/Alethio/ethstats-cli) - client app that extracts data from a node and sends it to the server 
+* [ethstats-network-server](https://github.com/Alethio/ethstats-network-server) - service that consumes the data received from the nodes through the client app 
+* [ethstats-network-dashboard](https://github.com/Alethio/ethstats-network-dashboard) - front end dashboard  
+
+## Installation
+
+The installation process is done through multiple steps. First the server needs to be installed with its dependencies and then for every node in the network the client app `ethstats-cli` is needed.
+
+#### Server
```

Comment:
I don't see where you make the link between the server and the Pantheon node. Can you clarify this point ?

Reply:
To bring more clarity, I added installation examples that includes the need of adding the `--server-url` flag and also `--client-url` if the Pantheon node is not running on the same host as `ethstats-cli`.